lib/miq_expression.rb

```diff
@@ -293,7 +294,7 @@ def self._to_ruby(exp, context_type, tz)
 
   def to_sql(tz = nil)
     tz ||= "UTC"
-    @pexp, attrs = preprocess_for_sql(@exp.deep_clone)
+    @pexp, attrs = preprocess_for_sql(exp.deep_clone)
```

Comment:
maybe for another PR, but why is `@pexp` an instance variable?  It seems that nobody else uses it in this file.

Reply:
@chessbyte Wondered about that. I could find no other reference to it in core. I changed it to a regular variable.